unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -193,63 +189,6 @@ export const EventDetails = ({
 
   return (
     <div>
-      <NextSeo
-        title={event.name}
-        description={`${truncateString(
-          removeMd(event.description, {
-            useImgAltText: false,
-          }),
-          650
-        )} 
-        Powered by Unlock Protocol`}
-        openGraph={{
-          title: event.title,
-          type: 'website',
-          url: eventUrl,
-          images: [
-            {
-              alt: event.title,
-              url: `${config.unlockApp}/og/event/${event.slug}`,
-            },
-          ],
-        }}
-        additionalMetaTags={[
-          {
-            property: 'fc:frame',
-            content: 'vNext',
-          },
-          {
-            name: 'fc:frame:image',
-            content: `${config.unlockApp}/og/event/${event.slug}`,
-          },
-          {
-            name: 'fc:frame:post_url',
-            content: `${config.unlockApp}/frames/event?p=${encodeURIComponent(
-              `${config.unlockApp}/frames/event/${event.slug}`
-            )}&s=${encodeURIComponent('{"view":"default"}')}`,
```

Comment:
So I wonder about all of this. Is it gone?

Reply:
nope!